modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -4909,6 +4909,14 @@
 		"issueKey": "LPD-54558",
 		"to": "getHelperUtilities(param#1#)"
 	},
+	{
+		"classNames": [
+			"LayoutCopyHelper"
+		],
+		"from": "copyLayout(Layout paramName, Layout paramName)",
+		"issueKey": "LPD-58415",
+		"to": "copyLayoutContent(param#0#, param#1#)"
+	},
```

Comment:
@regiCesar21 the copyLayout still exists and just one overload became copyLayoutContent? To be worth it validate the parameters before replacing? If no we can just remove the parameters of the pattern and will replace every case it finds.

If so, edit the tests as well as method() and null for parameters and just one case since won't trigger any message.